tests/scripts/uninstall-operator.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+if [[ "${SKIP_INSTALL}" == "true" ]]; then
+    echo "Skipping install: SKIP_INSTALL=${SKIP_INSTALL}"
+    exit 0
+fi
+
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+source ${SCRIPT_DIR}/.definitions.sh
+
+namespace=$1
+release_name=$2
+kubectl delete daemonsets --all -n $namespace || true
+kubectl delete deployments --all -n $namespace || true
+kubectl delete services --all -n $namespace || true
+kubectl delete pods --all -n $namespace || true
+kubectl delete namespace $namespace || true
```

Comment:
These `kubectl delete` commands are not needed, as the helm uninstall should delete all resources in the namespace. If desired, you can delete the namespace after uninstalling the helm chart.